covid19/src/components/DateChart.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+
+import { connect } from 'react-redux';
+
+import styles from '../styles/DateChart.module.css';
+
+import Paper from '@material-ui/core/Paper';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  LineSeries,
+  Legend,
+} from '@devexpress/dx-react-chart-material-ui';
+
+export class DateChart extends Component {
+    
+    render() {
```

Comment:
component does not hold any state nor has a lifecycle -> it can then be transformed into a functional component